p2p/pex/pex_reactor.go

```diff
@@ -109,6 +118,9 @@ func (r *PEXReactor) OnStart() error {
 	}
 
 	// return err if user provided a bad seed address
+	// NOTE: only if its an invalid address.
+	// If we simply fail to resovle a DNS name,
```

Comment:
I think it's fine to require host to resolve to something. See original issue: https://github.com/tendermint/tendermint/issues/1230 and the fix for it: https://github.com/tendermint/tendermint/issues/1230#issuecomment-374970140